applications/luci-app-usteer/htdocs/luci-static/resources/view/usteer/status.js

```diff
@@ -0,0 +1,264 @@
+							if (typeof hosts[macUp] !== 'undefined') {
+								if ((String(hosts[macUp]['ipaddrs'][0]).length>0) && (typeof hosts[macUp]['ipaddrs'][0] !== 'undefined')) {
+									foundname=hosts[macUp]['ipaddrs'][0];
+								}
+								if ((String(hosts[macUp]['name']).length>0) && (typeof hosts[macUp]['name'] !== 'undefined')) {
+									foundname=hosts[macUp]['name'];
+								}
+							}
+							hostl=hostl+ foundname+'&emsp;';
+				}
+			}		
+			compactconnectioninfo_table_entries.push([
+								'<nobr>'+wlan+'</nobr>', 
+								Localinfo[wlan]['ssid'],
+								Localinfo[wlan]['freq'],
+								Localinfo[wlan]['load'],
+								Localinfo[wlan]['n_assoc'],
+								hostl
+							]);
+		}
+		for(var wlan in Remoteinfo) {
+			var hostl=''
+			for(var mac in Clients) {
+				if (typeof Clients[mac] !== 'undefined') 
+					if (typeof Clients[mac][wlan] !== 'undefined') 
+					if (String(Clients[mac][wlan]['connected']).valueOf()==String("true").valueOf()) {
+							var foundname=mac;
+							var macUp=String(mac).toUpperCase()
+							if (typeof hosts[macUp] !== 'undefined') {
+								if ((String(hosts[macUp]['ipaddrs'][0]).length>0) && (typeof hosts[macUp]['ipaddrs'][0] !== 'undefined')) {
+									foundname=hosts[macUp]['ipaddrs'][0]; 
+								}
+								if ((String(hosts[macUp]['name']).length>0) &&	(typeof hosts[macUp]['name'] !== 'undefined')) {
+									foundname=hosts[macUp]['name'];
+								}
+							}
+							hostl=hostl+ foundname+'&emsp;';
+				}
+			}	
+			compactconnectioninfo_table_entries.push([
+								'<nobr>'+wlan+'</nobr>', 
+								Remoteinfo[wlan]['ssid'],
+								Remoteinfo[wlan]['freq'],
+								Remoteinfo[wlan]['load'],
+								Remoteinfo[wlan]['n_assoc'],
+								hostl
+							]);
+		}
+
+		cbi_update_table(compactconnectioninfo_table, compactconnectioninfo_table_entries, E('em', _('No data')));
+		body.appendChild(compactconnectioninfo_table);
+
+
+/////////////////////////
+		body.appendChild(
+			E('h3', 'Hearing map')
+		);
+		for(var mac in Clients) {
+			var maciphost='';
+			maciphost='Mac: '+mac;
+			var macUp=String(mac).toUpperCase()
+				if (typeof hosts[macUp] !== 'undefined') {
+					if (typeof hosts[macUp]['ipaddrs'] !== 'undefined') 
+						maciphost=maciphost+'&emsp;IP: '+hosts[macUp]['ipaddrs'];
+					if (typeof hosts[macUp]['name'] !== 'undefined') 
+						maciphost=maciphost+'&emsp;Host: '+hosts[macUp]['name'];
+				}
+			body.appendChild(
+				E('h4', maciphost)
+			);
+			var client_table = E('table', { 'class': 'table cbi-section-table' }, [
+				E('tr', { 'class': 'tr table-titles' }, [
+					E('th', { 'class': 'th' }, _('wlan')),
+					E('th', { 'class': 'th' }, _('connected')),
+					E('th', { 'class': 'th' }, _('signal'))
+				])
+			]);
+		
+			var client_table_entries =[];
+			for(var wlanc in Clients[mac]) {
+				client_table_entries.push([
+					'<nobr>'+wlanc+'</nobr>', 
+					(String(Clients[mac][wlanc]['connected']).valueOf()==String("true").valueOf()) ? "True" : "",
```

Comment:
Should be just
`Clients[mac][wlanc]['connected'] === true ? 'True' : ''`

Reply:
ok